server/lobbyconnection.py

```diff
@@ -594,10 +594,10 @@ def decodeUniqueId(self, serialized_uniqueid):
                 SerialNumber = str(machine.get('SerialNumber', 0)).encode() # BIOS serial number
                 VolumeSerialNumber = str(machine.get('VolumeSerialNumber', 0)).encode() # https://www.raymond.cc/blog/changing-or-spoofing-hard-disk-hardware-serial-number-and-volume-id/
```

Comment:
While we're at it, could we not give everybody looking at this code a hint on how to circumvent it? :P